lib/pipefy_message/consumer.rb

```diff
@@ -47,33 +48,15 @@ module ClassMethods
       # Merges default worker options with the hash passed as
       # an argument. The latter takes precedence.
       def options(opts = {})
```

Comment:
**suggestion (non-blocking):** I had actually agreed with you on this during our call haha But later I was thinking: since this is a module that'll be included in broader classes, what if the name "options" is used for something else? Could we maybe use consumer_options instead?

Reply:
Maybe I didn't understand your question, what do you mean by "broader classes"?

I mean... where will the consumer module be used besides a real consumer class?

ps: for the use case that we are aiming the name "options" works really well for me 😄